shipa/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#Add Helm Chart
+helm repo add shipa-charts https://shipa-charts.storage.googleapis.com
+helm repo update
+
+#Wait for the Cert Manager deployment to be done.
+kubectl wait --for=condition=available --timeout=300s deployment/cert-manager-webhook  -n cert-manager
+
+#Install Shipa
+helm upgrade --install shipa shipa-charts/shipa \
```

Comment:
So this installs shipa agent or something ? because Shipa as such is not open source right ?

Reply:
This is the Shipa Control Plane. It installs the free version by default or a user can supply a license.